sklearn/metrics/ranking.py

```diff
@@ -294,6 +294,11 @@ def _binary_clf_curve(y_true, y_score, pos_label=None, sample_weight=None):
     thresholds : array, shape = [n_thresholds]
         Decreasing score values.
     """
+    # Check to make sure y_true is valid
+    y_type = type_of_target(y_true)
+    if y_type != "binary":
```

Comment:
Also, in some of the other testing I'm getting the error: `ValueError: multiclass format is not supported`. I was under the impression that the `_binary_clf_curve` required 'binary' data.  Should it also be allowed to accept 'multiclass' data?